test/e2e/specs/all-features/navbar/base-config.spec.js

```diff
@@ -49,36 +49,40 @@ describe('Navbar ', function() {
             expect(count).toEqual(counter);
         });
     });
-    
-    
-    it('should open the profile card on click of My Profile link', function() {
-        var link = element(by.css('.dropdown-toggle'));    
+
+
+    it('should open the profile card on click of My Profile link', function(done) {
+        var link = element(by.css('.dropdown-toggle'));
         browser.wait(EC.elementToBeClickable(link), browser.params.defaultTimeout).then(function() {
             link.click();
             var profileLink = element(by.id('profile-link'));
-            browser.wait(EC.elementToBeClickable(profileLink), browser.params.defaultTimeout).then(function() {
-                return profileLink.click();
-            }).then(function () {
+            browser.wait(EC.elementToBeClickable(profileLink), browser.params.defaultTimeout);
+            chaisePage.clickButton(profileLink).then(function() {
```

Comment:
Was it the case that the button (at a specific location) couldn't be clicked which was causing this test to sometimes fail locally?

Reply:
Yes, this was one of the failing test cases. Now that you commented, I realized I should chain the previous click too (I changed and pushed the code).